### ISSUE TITLE
Couldn't fetch video info

### ISSUE THREAD

**Issue by fireflute** [JunkFood02/Seal#1834]
### Checklist

- [X] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

When i copy a link and try to download, this error shows up saying it couldn't fetch video info. I think this is a problem with yt-dlp as Seal just worked fine a couple of days ago

### Error reports

App version: 1.13.1-(F-Droid) (11312)
Device information: Android 6.0 (API 23)
Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]
Yt-dlp version: 2024.10.29.232854

URL: https://www.youtube.com/watch?v=26fnROvH1vo
WARNING: linker: /data/app/com.junkfood.seal-1/lib/arm64/libpython.so: unused DT entry: type 0x1d arg 0x4d
WARNING: linker: /data/data/com.junkfood.seal/no_backup/youtubedl-android/packages/python/usr/lib/libandroid-support.so: unused DT entry: type 0x1d arg 0x13f
WARNING: linker: /data/data/com.junkfood.seal/no_backup/youtubedl-android/packages/python/usr/lib/libpython3.11.so.1.0: unused DT entry: type 0x1d arg 0x9024
CANNOT LINK EXECUTABLE: cannot locate symbol "lockf" referenced by "/data/data/com.junkfood.seal/no_backup/youtubedl-android/packages/python/usr/lib/libpython3.11.so.1.0"...
page record for 0x7fae977010 was not found (block_size=128)

**Comment by bashonly** [comment]
According to https://github.com/deniscerri/ytdlnis/issues/590#issuecomment-2414803074, the [latest version of the youtubedl-android library](https://github.com/yausername/youtubedl-android/releases/tag/0.17.1) requires Android >=7.0.

Seal version 1.13.1 depends on this latest version of that library, since it provides support for Python >=3.9, which is needed to run the latest nightly/master versions of yt-dlp (and very soon the latest stable version of yt-dlp).

The tl;dr summary is that Seal version 1.13.1 may not work on Android 6.0 devices.

You could try downgrading Seal to version 1.13.0 and downgrading yt-dlp to stable@2024.10.22 and see if that works on your Android 6.0 device. But you won't be able to update yt-dlp beyond stable@2024.10.22 if you are using Seal version 1.13.0

**Comment by fireflute** [comment]
I suspected about that. Thanks for the reply.
